### ISSUE TITLE
how can I know the true index ?

### ISSUE THREAD

**Issue by bilipa** [vana77/Market-1501_Attribute#3]
In the mat file, the market_attribute train shape is 751x27.
How can I know the first row of it is which image ?
The bounding box of train folder have 751 identity, but i do not know how to map the label to image

**Comment by olimoser** [comment]
so I think it is like this:

...
train/1496_c6s3_094367_04.jpg
train/1496_c6s3_094367_06.jpg
train/1500_c1s6_014121_04.jpg
...

An identity is defined by the first part of the filename (here: 1496, 1500)
I took all images from train and extracted the first part.
I took a unique set of this.
I sorted it.
Then i have a mapping like { 0: "0002", 1: "0007", 2: "0010" ....}

I manually did a sanity check of ~20 samples comparing age and gender with this mapping and it seems correct ...
